settings.py

```diff
@@ -2,10 +2,16 @@
 import dj_database_url  # type: ignore
 import boto3
 
+# Get secret and access key
+AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
+
 # Get db paasword from AWS SSM
-session = boto3.Session(profile_name="default")
-ssm = session.client("ssm")
+session = boto3.Session(
```

Comment:
What changed that we need a key pair instead of just a profile name?

Reply:
The initial version was not deployed, worked locally. On the lambda instance the has these key pairs in its default environment so we can reference them.

Reply:
Reference here: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html